formats/json-tests/commonTest/src/kotlinx/serialization/SerializersLookupTest.kt

```diff
@@ -52,6 +52,21 @@ class SerializersLookupTest : JsonTestBase() {
         assertSerializedWithType("""["a","b","c"]""", myArr)
     }
 
+    @Test
+    fun testUnsigned() = noLegacyJs {
+        assertSame(UByte.serializer(), serializer<UByte>())
+        assertSame(UShort.serializer(), serializer<UShort>())
+        assertSame(UInt.serializer(), serializer<UInt>())
+        assertSame(ULong.serializer(), serializer<ULong>())
+    }
+    @Test
+    @OptIn(ExperimentalUnsignedTypes::class)
+    fun testUnsignedArrays() {
```

Comment:
Is this test working on legacy JS? If so, why previous doesn't?

Reply:
Same problem as in `testCustomGeneric`:
```
TypeError: tmp$.serializer is not a function
  ...
```